sanity/schemaTypes/csr.ts

```diff
@@ -0,0 +1,73 @@
+import { defineField, defineType } from "sanity";
+import { BlockElementIcon } from "@sanity/icons";
+
+export const CSRType = defineType({
+    name: "csr",
+    title: "CSR",
+    type: "document",
+    icon: BlockElementIcon,
+    fields: [
+        defineField({
+            name: "title",
+            title: "Title",
+            type: "string",
+            validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+            name: "titleColor",
+            title: "Title Color",
+            type: "color",
+            validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+            name: "subtitle",
+            title: "Subtitle",
+            type: "string",
+            validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+            name: "description",
+            title: "Description",
+            type: "array",
+            of: [{
+                type: 'block',
+                marks: {
+                    annotations: [],
+                },
+            }],
+            validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+            name: "images",
+            title: "Images",
+            type: "array",
+            of: [{ type: "image" }],
+            options: {
+                layout: "grid",
+            },
+            validation: (Rule) => Rule.required(),
+        }),
+        defineField({
+            name: "position",
+            title: "Position",
+            type: "number",
+            validation: (Rule) => Rule.required(),
```

Comment:
please add validation for duplicate position

Reply:
roger that!